index.html

```diff
@@ -19156,25 +19168,25 @@ <h2 id="AudioWorklet-Sequence">
             <a>AudioWorkletProcessor</a>.
           </p>
           <ol>
-            <li>In the main scope, <code>window.audioWorklet</code> is
-            requested to import a script. No <a>AudioWorkletGlobalScope</a>s
-            exist yet, so the script is fetched and added to the Worklet module
-            responses map.
-            </li>
             <li>An <a>AudioContext</a> is created.
             </li>
-            <li>An <a>AudioWorkletGlobalScope</a> is created in association
-            with the context's audio rendering thread. This is the global scope
-            in which <a>AudioWorkletProcessor</a> class definitions will be
-            evaluated.
+            <li>In the main scope, <code>context.audioWorklet</code> is
+            requested to add a script module.
+            </li>
+            <li>Since none exists yet, a new <a>AudioWorkletGlobalScope</a> is
+            created in association with the context's audio rendering thread.
+            This is the global scope in which <a>AudioWorkletProcessor</a>
+            class definitions will be evaluated.
             </li>
-            <li>As part of the global scope's initialization, the set of
-            imported scripts is run, including the one that was previously
-            imported.
+            <li>The imported script is run in the newly created global scope.
             </li>
             <li>As part of running the imported script, an
             <a>AudioWorkletProcessor</a> is registered under the key
             <code>"Custom1"</code> within the <a>AudioWorkletGlobalScope</a>.
+            This populates maps both in the global scope and in the
+            <a>AudioContext</a>.
+            </li>
+            <li>The promise for the addModule() call is resolved.
             </li>
             <li>In the main scope, an <a>AudioWorkletNode</a> is created using
             the key <code>"Custom1"</code> along with an <code>opts</code>
```

Comment:
Missed this before too, but "opts" dictionary should probably be a link to something so I can actually look up exactly what this is.  This can be addressed in a different PR.

Reply:
Same as above